ass/libass.py

```diff
@@ -1,7 +1,7 @@
-# pip install clang ctypeslib2
+# pip install clang ib2
 # clang2py -o libass.py -c -d -l libass.so ass.h
 
-import ctypes
+from ctypes import *
```

Comment:
Why?

Reply:
We need that for `CFUNCTYPE` and other `cytpes` functions and fields. What's the problem?

Reply:
Polluting global namespace with tons of stuff, unfriendly ide, unfriendly grep

Reply:
I see, I will change that!